static/js/util/validation.js

```diff
@@ -71,6 +73,10 @@ export function educationValidation(profile) {
   ];
   if ( !_.isEmpty(profile.education) ) {
     let requiredFields = handleNestedValidation(profile, nestedKeys, 'education');
+    requiredFields = requiredFields.filter(([, index, key]) =>
+      // don't require field of study for high school students
+      !(key === 'field_of_study' && _.get(profile, ['education', index, 'degree_name']) === HIGH_SCHOOL)
+    );
```

Comment:
nice! this is exactly what I was hoping this refactor (to `validation.js`) would enable, easily inserting logic for more specific validations.